From 63711d1f070acbe38792e6310201182892494dc5 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Wed, 10 Sep 2025 14:00:23 +0200
Subject: [PATCH 20/22] mvebu-armada8k: add nvme boot target

armada 8k platforms support pci and can boot from nvme if present.
Add nvme as a distro-boot target.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 include/configs/mvebu_armada-8k.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mvebu_armada-8k.h b/include/configs/mvebu_armada-8k.h
index 5debd9117c..bcb699031a 100644
--- a/include/configs/mvebu_armada-8k.h
+++ b/include/configs/mvebu_armada-8k.h
@@ -35,6 +35,7 @@
 	func(MMC, mmc, 1) \
 	func(MMC, mmc, 0) \
 	func(USB, usb, 0) \
+	func(NVME, nvme, 0) \
 	func(SCSI, scsi, 0) \
 	func(PXE, pxe, na) \
 	func(DHCP, dhcp, na)
-- 
2.51.0

